src/Compilers/CSharp/Portable/Syntax/SyntaxNormalizer.cs

```diff
@@ -264,10 +270,73 @@ private int LineBreaksAfter(SyntaxToken currentToken, SyntaxToken nextToken)
             return 0;
         }
 
+        private static bool All<TNode>(SyntaxList<TNode> items, Func<TNode, bool> predicate)
+            where TNode : SyntaxNode
+        {
+            bool result = true;
+            for (int i = 0; i < items.Count; i++)
+            {
+                result &= predicate(items[i]);
+            }
+
+            return result;
+        }
+
+        private static bool IsAutoAccessorList(SyntaxToken token)
+            => token.Parent is AccessorListSyntax accessorList &&
+               All(accessorList.Accessors, a => a.Body == null);
+
+        private static bool IsAutoAccessorList(SyntaxNode? node)
+        {
+            if (node?.Parent is AccessorListSyntax accessorList)
+            {
+                return All(accessorList.Accessors, a => a.Body == null);
+            }
+
+            return false;
```

Comment:
same cleanup here :)

Reply:
It's night here. Will do, not now.
It's a pity i did not get it before :) The tests take soooo long

Reply:
You can run them locally in VS :)

Reply:
I was hoping in the so called power of the Cloud ;-) At the very end, running them locally is faster LOL